pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -53,6 +53,8 @@ let
       ./qtbase.patch.d/0009-qtbase-qtpluginpath.patch
       ./qtbase.patch.d/0010-qtbase-assert.patch
       ./qtbase.patch.d/0011-fix-header_module.patch
+    ] ++ lib.optional (stdenv.isDarwin && stdenv.isAarch64) [
+      ./qtbase.patch.d/0002-qtbase-mac-arm64.patch
```

Comment:
Could we fix this universally and apply the patch all the time?

Reply:
Homebrew took the universal way:

https://raw.githubusercontent.com/Homebrew/formula-patches/9dc732/qt/qt-split-arch.patch